<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager_window</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>1003</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="employee_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>settings.png</normaloff>settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>73</width>
      <height>61</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Employee</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="add_guest_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>-10</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>traveler-with-a-suitcase.png</normaloff>traveler-with-a-suitcase.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>73</width>
      <height>61</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Add Guest</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="check_out_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-10</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>exit.png</normaloff>exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>73</width>
      <height>61</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Check Out</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>employee_button</zorder>
   <zorder>add_guest_button</zorder>
   <zorder>check_out_button</zorder>
  </widget>
  <widget class="QWidget" name="add_guest_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>981</width>
     <height>421</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="Guest_full_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px black solid</string>
    </property>
    <property name="placeholderText">
     <string>                Guest Full Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="adults_label">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>10</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adults</string>
    </property>
   </widget>
   <widget class="QLabel" name="children_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>children</string>
    </property>
   </widget>
   <widget class="QComboBox" name="room_view_box">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sea</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pool</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Street</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="room_view_label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Room View</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="guest_phone_number">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px black solid</string>
    </property>
    <property name="placeholderText">
     <string>            Guest Phone Number</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nights_box">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nights_label">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>78</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nights</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="guest_email">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px black solid</string>
    </property>
    <property name="placeholderText">
     <string>                    Guest Email</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="adults_box">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="children_box">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="check_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 255, 0)</string>
    </property>
    <property name="text">
     <string>CHECK</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="room_type_box">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="room_type_lable">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Room Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_price">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total price : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="reserve_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 255, 0)</string>
    </property>
    <property name="text">
     <string>Reserve</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 255, 0)</string>
    </property>
    <property name="text">
     <string>Confirm Reservation</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="total_price_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Room Number :</string>
    </property>
   </widget>
   <zorder>confirm_button</zorder>
   <zorder>Guest_full_name</zorder>
   <zorder>adults_label</zorder>
   <zorder>children_label</zorder>
   <zorder>room_view_box</zorder>
   <zorder>room_view_label</zorder>
   <zorder>guest_phone_number</zorder>
   <zorder>nights_box</zorder>
   <zorder>nights_label</zorder>
   <zorder>guest_email</zorder>
   <zorder>adults_box</zorder>
   <zorder>children_box</zorder>
   <zorder>check_button</zorder>
   <zorder>room_type_box</zorder>
   <zorder>room_type_lable</zorder>
   <zorder>total_price</zorder>
   <zorder>reserve_button</zorder>
   <zorder>total_price_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
